Mumble fails to build

## Issue description

Hydra log:
https://hydra.nixos.org/build/92486838
```
In file included from BanEditor.cpp:36:0:
Global.h:142:11: error: expected ',' or '...' before '(' token
 #define g (*Global::g_global_struct)
           ^
compilation terminated due to -Wfatal-errors.
```

This might be related to this upstream issue:
https://github.com/mumble-voip/mumble/issues/3617

I tried to apply the PR fixing the above issue but the patch is on master and therefore can't be applied directly to 1.2.19 without conflicts

### Steps to reproduce

`nix-build -A mumble` on latest nixpkgs master

--- a/pkgs/applications/networking/mumble/default.nix
+++ b/pkgs/applications/networking/mumble/default.nix
@@ -1,5 +1,5 @@
 { stdenv, fetchurl, fetchFromGitHub, fetchpatch, makeWrapper, pkgconfig
-, qt4, qmake4Hook, qt5, avahi, boost, libopus, libsndfile, protobuf, speex, libcap
+, qt4, qmake4Hook, qt5, avahi, boost, libopus, libsndfile, protobuf3_6, speex, libcap
 , alsaLib, python
 , jackSupport ? false, libjack2 ? null
 , speechdSupport ? false, speechd ? null
@@ -22,7 +22,10 @@ let
     nativeBuildInputs = [ pkgconfig python ]
       ++ { qt4 = [ qmake4Hook ]; qt5 = [ qt5.qmake ]; }."qt${toString source.qtVersion}"
       ++ (overrides.nativeBuildInputs or [ ]);
-    buildInputs = [ boost protobuf avahi ]
+
+    # protobuf is freezed to 3.6 because of this bug: https://github.com/mumble-voip/mumble/issues/3617
+    # this could be reverted to the latest version in a future release of mumble as it is already fixed in master
+    buildInputs = [ boost protobuf3_6 avahi ]
       ++ { qt4 = [ qt4 ]; qt5 = [ qt5.qtbase ]; }."qt${toString source.qtVersion}"
       ++ (overrides.buildInputs or [ ]);
 

